ansible/pbTestScripts/vagrantPlaybookCheck.sh

```diff
@@ -211,32 +212,27 @@ startVMPlaybook()
 	# The BUILD_ID variable is required to stop Jenkins shutting down the wrong VMS 
 	# See https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1287#issuecomment-625142917
 	BUILD_ID=dontKillMe vagrant up
-	# FreeBSD12 / Debian10 uses an rsync shared folder type- required to get hosts.tmp from VM
-	if [[ "$OS" == "FreeBSD12" || "$OS" == "Debian10" ]]; then
-               vagrant rsync-back
-	fi
-	# Generate hosts.unx file for Ansible to use, remove prior hosts.unx if there
-	[[ -f playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx ]] && rm playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx
-	cat playbooks/AdoptOpenJDK_Unix_Playbook/hosts.tmp | tr -d \\r | sort -nr | head -1 > playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx && rm playbooks/AdoptOpenJDK_Unix_Playbook/hosts.tmp
-	local vagrantIP=$(cat playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx)
+
+	rm -f playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx
+	echo "[127.0.0.1]:$(vagrant  port | grep host | awk '{ print $4 }')" >> playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx
```

Comment:
double whitespace after vagrant

Reply:
It's definitely $4 we want?

Reply:
Yep, definitely! Tested here https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/724/ :-)